x-pack/legacy/plugins/spaces/public/space_selector/index.tsx

```diff
@@ -15,14 +15,14 @@ import React from 'react';
 import { render, unmountComponentAtNode } from 'react-dom';
 import { SpaceSelector } from './space_selector';
 
-import { start as spacesNPStart } from '../../legacy';
+import { start as spacesNPStart } from '../legacy';
 
 const module = uiModules.get('spaces_selector', []);
 module.controller('spacesSelectorController', ($scope: any) => {
   $scope.$$postDigest(async () => {
     const domNode = document.getElementById('spaceSelectorRoot');
 
-    const { spacesManager } = await spacesNPStart;
+    const { spacesManager } = spacesNPStart;
```

Comment:
__optional nit:__ looks like we can remove `async` from the `postDigest` callback function now.